### ISSUE TITLE
Allocated is not divided by OperationsPerInvoke

### ISSUE THREAD

**Issue by benaadams** [dotnet/BenchmarkDotNet#797]
I get this output for https://gist.github.com/benaadams/294cbd41ec1179638cb4b5495a15accf
```
    Method | Categories |      Mean |     Error |    StdDev | Scaled | Allocated |
---------- |----------- |----------:|----------:|----------:|-------:|----------:|
      Linq |        Any | 16.718 ns | 0.0615 ns | 0.0480 ns |   1.00 |     320 B |
 ValueLinq |        Any |  8.241 ns | 0.0286 ns | 0.0253 ns |   0.49 |       0 B |
           |            |           |           |           |        |           |
      Linq |      Count | 69.538 ns | 0.3562 ns | 0.2781 ns |   1.00 |      32 B |
 ValueLinq |      Count | 35.665 ns | 0.1095 ns | 0.0970 ns |   0.51 |       0 B |
```

However the "Any" category has 10 OperationsPerInvoke so I think it should also show `32B` rather than `320B`

**Comment by adamsitnik** [comment]
Hi Ben!

Thanks for the bug report, it has already been fixed in #724.

The fix will be a part of `0.11.0`, but before we ship it to nuget.org, you can get can it today from our CI feed:

```
<packageSources>
  <add key="bdn-ci" value="https://ci.appveyor.com/nuget/benchmarkdotnet" />
</packageSources>
```

**Comment by benaadams** [comment]
So fast! 😄
